runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/components/BasicTextField.tsx

```diff
@@ -64,19 +74,24 @@ const BasicTextField: FC<BasicTextFieldProps> = ({
     }, [debouncedValue]);
 
     return (
-        <TextField
-            {...params}
-            value={currentValue}
-            fullWidth
-            variant="outlined"
-            disabled={isFieldDisabled}
-            required={props.required}
-            label={props.label}
-            onChange={(event) => handleChange(event.target.value)}
-            InputLabelProps={{ shrink: true }}
-            error={Boolean(errors) || Boolean(props.rawErrors)}
-            helperText={errors ? errors : null}
-        />
+        <TooltipTextfieldWrapper>
+            <TextField
+                {...params}
+                value={currentValue}
+                fullWidth
+                variant="outlined"
+                disabled={isFieldDisabled}
+                required={props.required}
+                label={props.label}
+                onChange={(event) => handleChange(event.target.value)}
+                InputLabelProps={{ shrink: true }}
+                error={Boolean(errors) || Boolean(props.rawErrors)}
+                helperText={errors ? errors : null}
+            />
+            <If condition={Boolean(props.options?.info)}>
+                <InfoButtonTooltip message={props.options?.info}/>
```

Comment:
redundant ?

Reply:
questionmark

Reply:
We need it - without that tooltip hasn't existed in TextFields without autocomplete (it was bug when tooltip dissapeared from StartProcess -> output).